src/foremast/utils/pipelines.py

```diff
@@ -115,3 +115,36 @@ def normalize_pipeline_name(name=''):
     for bad in '\\/?%#':
         normalized_name = normalized_name.replace(bad, '_')
     return normalized_name
+
+
```

Comment:
General placement of the util. 

Would you consider this a pipeline utility or kayenta/canary util?